tests/Application/Actions/Person/GetTest.php

```diff
@@ -156,7 +160,55 @@ public function testSuccessWithNoStripeBalances(): void
         $this->assertObjectNotHasAttribute('gbp', $payload->cash_balance);
     }
 
-    public function testSuccessWithNonAutomaticallyReconcileStripeBalances(): void
+    public function testSuccessWithPendingTipAndNoBalances(): void
+    {
+        $person = $this->getInitialisedPerson(true);
+
+        $app = $this->getAppInstance();
+        /** @var Container $container */
+        $container = $app->getContainer();
+
+        $personRepoProphecy = $this->prophesize(PersonRepository::class);
+        $personRepoProphecy->find(static::$testPersonUuid)
```

Comment:
Not sure how exactly I've got a PHPStan thing in my PHPStorm now. When I read this it pops up complaining that the static usage is unsafe. Afaik there's no reason for it here over the simpler `self`.
```suggestion
        $personRepoProphecy->find(self::$testPersonUuid)
```